{"name": "测试输入：{\"grant_type\":\"password\",\"username\":\"\",\"password\":\"\",\"imageCode\":\"9527\"}", "status": "passed", "parameters": [{"name": "inData", "value": "'{\"grant_type\":\"password\",\"username\":\"\",\"password\":\"\",\"imageCode\":\"9527\"}'"}, {"name": "respData", "value": "'{\"error\": \"invalid_grant\",\"error_description\": \"Bad credentials\"}'"}], "start": 1619077452068, "stop": 1619077452255, "uuid": "e32ca39c-5d63-4e78-9eb5-956b206d13c3", "historyId": "b071b0ed921596161105b1ef4782fbe1", "testCaseId": "eec049e373b40a29ff5c4ae98ab4d8f6", "fullName": "test_login.TestLogin#test_login_s2", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "登录模块"}, {"name": "epic", "value": "TP系统"}, {"name": "story", "value": "登录接口"}, {"name": "tag", "value": "异常账户登陆"}, {"name": "tag", "value": "Login(order=1)"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-5R42TFK"}, {"name": "thread", "value": "13208-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}